.vscode/settings.json

```diff
@@ -11,6 +11,7 @@
     "rust-analyzer.rustfmt.extraArgs": [
       "+nightly"
     ],
+    "workbench.colorCustomizations": {},
```

Comment:
I'm guessing this was left in in error? Can we remove it?

Reply:
Done!